quickstart/targets.go

```diff
@@ -0,0 +1,79 @@
+package quickstart
+
+import (
+	"fmt"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/huh"
+	"github.com/pkg/errors"
+	"github.com/speakeasy-api/openapi-generation/v2/pkg/generate"
+	"github.com/speakeasy-api/sdk-gen-config/workflow"
+	"github.com/speakeasy-api/speakeasy/charm"
+)
+
+func targetBaseForm(inputWorkflow *workflow.Workflow) (*State, error) {
+	var targetName, targetType, sourceName, outputLocation string
+	if _, err := tea.NewProgram(charm.NewForm(huh.NewForm(
+		huh.NewGroup(
+			huh.NewInput().
+				Title("A name for this target:").
+				Placeholder("unique name across this workflow").
+				Validate(func(s string) error {
+					if _, ok := inputWorkflow.Targets[s]; ok {
+						return fmt.Errorf("a source with the name %s already exists", s)
+					}
+					return nil
+				}).
+				Inline(true).
+				Prompt(" ").
+				Value(&targetName),
+			huh.NewSelect[string]().
+				Title("What target would you like to generate:").
+				Description("Choose from this list of supported generation targets. \n").
+				Options(huh.NewOptions(getSupportedTargets()...)...).
+				Value(&targetType),
+			huh.NewSelect[string]().
+				Title("What source would you like to generate this target from:").
+				Description("Choose from this list of existing sources \n").
+				Options(huh.NewOptions(getSourcesFromWorkflow(inputWorkflow)...)...).
+				Value(&sourceName),
+			huh.NewInput().
+				Title("Provide an output location for your generation target (OPTIONAL):").
```

Comment:
Can we add "Defaults to the current directory" or whatever as a description? Also Maybe change it to `Optionally provide...`